src/Audio.c

```diff
@@ -256,17 +322,48 @@ static void Sounds_Start(void) {
 	AudioBackend_LoadSounds();
 }
 
+static void Sounds_FreeCustom(struct SoundArray* group) {
+	for (int i = 0; i < group->count; i++) {
+		if (group->sounds) {
+			struct AudioChunk chunk = group->sounds[i].chunk;
+			if (chunk.data) {
+				Mem_Free(chunk.data);
+			}
+		}
+	}
+	group->count = 0;
+}
+

```

Comment:
As per my previous point about allocation, string buffers of strings inside `group->soundNames` are never freed.